cnmodel/data/connectivity.py

```diff
@@ -55,7 +55,7 @@
 
 """
 
-add_table_data('mouse_convergence', row_key='pre_type', col_key='post_type', 
+add_table_data('convergence', row_key='pre_type', col_key='post_type', 
```

Comment:
It's fine for the mouse and guineapig convergence tables to have the same name; we will access them like:

    data.get('convergence', species='mouse', pre_type='sgc', post_type='bushy')

Reply:
agreed - forgot that the whole set of parameters is the "key" to the data